tests/e2e/e2e_test.go

```diff
@@ -471,8 +471,11 @@ var _ = Describe("odoe2e", func() {
 			cmpList := runCmd("odo list")
 			Expect(cmpList).NotTo(ContainSubstring("nodejs"))
 
-			// TODO: `odo project delete` once implemented
-			runCmd("oc delete project " + projName)
+			runCmd("odo project delete " + projName)
+			// Wait 15 secs to see the changes in list of projects post deletion of a project
+			time.Sleep(15 * time.Second)
```

Comment:
See other comment regarding adding a timer... This adds unnecessary wait.

Reply:
Ack Done